canopen/profiles/p402.py

```diff
@@ -249,7 +249,7 @@ def _init_tpdo_values(self):
             if tpdo.enabled:
                 tpdo.add_callback(self.on_TPDOs_update_callback)
                 for obj in tpdo:
-                    logger.debug('Configured TPDO: {0}'.format(obj.index))
+                    logger.debug('Configured TPDO: %s', obj.index)
```

Comment:
This should be formatted as hex `0x%04X` or using `pretty_index()` for consistency.

Reply:
Nice catch! Although `pretty_index()` will be called even if logging is not printed. In a way it negates the reason for having the legacy "%s" syntax. All in all, I think having it outweighs the downside.

Reply:
Right, then let's ignore this case.

Reply:
Sorry, that was a misunderstanding.  I thought we agreed on avoiding the function call overhead because logging will very likely not even use the value?

Reply:
Reverted